tools/resourcedocsgen/pkg/lookup.go

```diff
@@ -0,0 +1,196 @@
+	"github-serverless-webhook":           PackageCategoryVCS,
+	"gitlab":                              PackageCategoryVCS,
+	"hcloud":                              PackageCategoryCloud,
+	"istio-helm":                          PackageCategoryInfrastructure,
+	"jaeger-helm":                         PackageCategoryMonitoring,
+	"kafka":                               PackageCategoryInfrastructure,
+	"keycloak":                            PackageCategoryInfrastructure,
+	"kong":                                PackageCategoryInfrastructure,
+	"kubernetes":                          PackageCategoryCloud,
+	"libvirt":                             PackageCategoryUtility,
+	"linode":                              PackageCategoryCloud,
+	"mailgun":                             PackageCategoryInfrastructure,
+	"minio":                               PackageCategoryInfrastructure,
+	"mongodbatlas":                        PackageCategoryDatabase,
+	"mysql":                               PackageCategoryDatabase,
+	"newrelic":                            PackageCategoryMonitoring,
+	"nginx-ingress-controller-helm":       PackageCategoryNetwork,
+	"nomad":                               PackageCategoryInfrastructure,
+	"ns1":                                 PackageCategoryNetwork,
+	"okta":                                PackageCategoryInfrastructure,
+	"onelogin":                            PackageCategoryInfrastructure,
+	"openstack":                           PackageCategoryCloud,
+	"opsgenie":                            PackageCategoryInfrastructure,
+	"pagerduty":                           PackageCategoryInfrastructure,
+	"postgresql":                          PackageCategoryDatabase,
+	"prometheus-helm":                     PackageCategoryMonitoring,
+	"rabbitmq":                            PackageCategoryInfrastructure,
+	"rancher2":                            PackageCategoryInfrastructure,
+	"random":                              PackageCategoryUtility,
+	"rke":                                 PackageCategoryInfrastructure,
+	"shipa":                               PackageCategoryCloud,
+	"signalfx":                            PackageCategoryMonitoring,
+	"snowflake":                           PackageCategoryInfrastructure,
+	"splunk":                              PackageCategoryInfrastructure,
+	"spotinst":                            PackageCategoryInfrastructure,
+	"sumologic":                           PackageCategoryMonitoring,
+	"tls":                                 PackageCategoryUtility,
+	"vault":                               PackageCategoryInfrastructure,
+	"venafi":                              PackageCategoryInfrastructure,
+	"vsphere":                             PackageCategoryCloud,
+	"wavefront":                           PackageCategoryMonitoring,
+	"yandex":                              PackageCategoryCloud,
+}
+
+// TODO[pulumi/pulumi#7813]: Remove this lookup once display name is available in
```

Comment:
[This issue](https://github.com/pulumi/pulumi/issues/7813) was closed in Aug, 2021.

Can we create an issue here to clean this up?

Reply:
https://github.com/pulumi/registry/issues/4672